dev/Scroller/Scroller.idl

```diff
@@ -1,5 +1,14 @@
 ﻿[WUXC_VERSION_PREVIEW]
```

Comment:
In other place, we use MU_XC_NAMESPACE.ContentOrientation. does it match with WUXC_VERSION_PREVIEW

Reply:
Yes, in Microsoft.UI.Xaml.idl you'll see that Scroller\Scroller.idl is included for the MU_XC_NAMESPACE.
Repeater\ItemsRepeater.idl and ItemsRepeater's AnimationContext do the same for instance.